docs/reference/query.rst

```diff
@@ -163,18 +163,24 @@ and day are optional. If you leave out the day, for example, you will get
 matches for the whole month.
 
 You can also use relative dates to the current time.
-Relative dates begin by the ``@`` character, followed by an optional ``+`` or
-``-`` sign that will increment or decrement now, followed by the time quantity
-that will be represented as an integer followed by either ``d`` for days, ``w``
-for weeks, ``m`` for months and finally ``y`` for year.
+A relative date begins by ``@`` and has to follow the pattern_dq format
+``@([+|-]?)(\d+)([y|m|w|d])``
+- ``@`` indicates it's a date relative to now()
+- the optional ``+`` or ``-`` sign, which defaults to ``+`` will increment or
+decrement now() by a certain quantity
+- that quantity can be expressed in days, weeks, months or years respectively
```

Comment:
Maybe, just for completeness' sake (and in line with a previous comment), maybe add a bullet point for the numbers so each of 4 parts of the relative date is explained?